UIKitDZ/UIKitDZ/Extensions/UIColor + extension.swift

```diff
@@ -0,0 +1,33 @@
+// UIColor + extension.swift
```

Comment:
```suggestion
// UIColor+Extension.swift
```
В названии файла все поправь тоже